### PR TITLE
Fix git-dir test during install

### PR DESCRIPTION
**Pull request by gabyx**

### CODE REVIEW

File: tests/test-windows.sh [hunk 1]
```diff
@@ -31,6 +31,7 @@ sed -i -E 's|download_file.*"install.sh"|cp -f /var/lib/githooks/install.sh|' "$
 # Change the base template so we can pass in the hook name and accept flags
 # shellcheck disable=SC2016
 sed -i -E 's|HOOK_NAME=.*|HOOK_NAME=\${HOOK_NAME:-\$(basename "\$0")}|' "$ROOT_DIR"base-template.sh &&
+    sed -i -E 's|echo.*Hook not run inside a git repository.*|CURRENT_GIT_DIR=".git"|' /var/lib/githooks/base-template.sh /var/lib/githooks/install.sh &&
```
Comment (rycus86) [inline tests/test-windows.sh]: Is this a related change? 🤔

Comment (gabyx) [reply tests/test-windows.sh]: This was a left over from the #78. Forgot to put that also here. Same replacements as in `exec-tests.sh`

**Comment by rycus86** [comment]
Thanks a lot!
This also seems to have fixed the coverage test. 🎉

**Comment by gabyx** [comment]
Oh dear, how lucky? Strange strange...
